great_expectations/data_context/data_context.py

```diff
@@ -465,32 +465,41 @@ def get_batch(self, batch_kwargs, expectation_suite_name, data_asset_type=None,
 
         Args:
             batch_kwargs: the batch_kwargs to use; must include a datasource key
-            expectation_suite_name: the name of the expectation_suite to get
+            expectation_suite_name: The ExpectationSuite or the name of the expectation_suite to get
             data_asset_type: the type of data_asset to build, with associated expectation implementations. This can
```

Comment:
Any opinions on if we should rename the `expectation_suite_name` parameter on `get_batch` here or in a future breaking release?